src/typescript/frontend/src/app/dev/verify_api_keys/page.tsx

```diff
@@ -52,6 +53,13 @@ const VerifyApiKeys = async () => {
     throw new Error(`Couldn't fetch the price feed on the server. ${msg}`);
   }
 
+  // Check that CoinMarketCap API key works
```

Comment:
- [ ] Try CoinGecko, ideally without an API key if possible
- [ ] Mark as resolved this comment if able to use it